pkgs/dart_mcp_server/lib/src/mixins/dash_cli.dart

```diff
@@ -179,6 +179,13 @@ base mixin DashCliSupport on ToolsSupport, LoggingSupport, RootsTrackingSupport
     );
   }
 
+  static final List<Tool> allTools = [
```

Comment:
nit: consider looping through this list in the initialize method so that it is less likely a new tool is added to initialize and not added here. You'd have to store that tools with their associated callback here.

Reply:
> it is less likely a new tool is added to initialize and not added here.

Yeah, I was a little worried about the list of tools getting out of date. I re-wrote this PR to instead spin up the MCP server and ask it for the list of tools; this should be much less maintenance than keeping separate lists of tools in the src code.